### ISSUE TITLE
[clsim] sign compare warning (Trac #1662)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1654]
```text
/home/dschultz/Documents/combo/trunk/src/clsim/private/clsim/tabulator/Axis.cxx:142:26:
    warning: comparison between signed and unsigned integer expressions [-
    Wsign-compare]
for (unsigned i = 0; i < power_-1; i++)
~~^~~~~~~~~~
```

Migrated from https://code.icecube.wisc.edu/ticket/1662
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "{{{\n/home/dschultz/Documents/combo/trunk/src/clsim/private/clsim/tabulator/Axis.cxx:142:26:\n    warning: comparison between signed and unsigned integer expressions [-\n    Wsign-compare]\nfor (unsigned i = 0; i < power_-1; i++)\n~~^~~~~~~~~~\n}}}", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[clsim] sign compare warning", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2016-04-26T20:30:18", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170327T19:35:45: 
* claudio.kopper commented:

> The tabulator was written by Jakob, assigning to him.


* claudio.kopper changed owner from "claudio.kopper" to "jvansanten"
* claudio.kopper changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20170327T19:40:31: 
* jvansanten commented:

> In changeset 154406:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="154406"
> Fix #1654
> ```


* jvansanten changed status from "assigned" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas commented:

> Milestone IceSim 5.1 deleted


* olivas changed milestone from "IceSim 5.1" to ""
